code-reviews/recipes/CSharp.md

```diff
@@ -100,3 +100,16 @@ In addition to the [Code Review Checklist](../process-guidance/reviewer-guidance
 * [ ] Does this code properly validate arguments sanity (i.e. [CA1062](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1062))? Consider leveraging extensions such as [Ensure.That](https://github.com/danielwertheim/Ensure.That)
 * [ ] Does this code include telemetry ([metrics, tracing](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) and [logging](https://serilog.net/)) instrumentation?
 * [ ] Does this code leverage the [options design pattern](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1) by using classes to provide strongly typed access to groups of related settings?
+* [ ] Are constants used in the main class? Or if this is used across files/classes, is there a static class for the constants?
```

Comment:
What do you mean by the first question (constants in main class)?

Reply:
ah, instead of using just raw strings (e.g. for routes or paths) in code :). Let me see if there is a better way to phrase this.

Reply:
Should be more clear now, thank you!